store.html

```diff
@@ -0,0 +1,122 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Karma</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/store.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+
+  <body>
+<img src="img/karma-logo.svg" alt="karma-logo" class="logo" />
+   
+      <div class="content">
+      <div class="navbar">
+        <a style="font-weight: bold" href="index.html">Meet Karma</a>
+        <a href="#">How it Works</a>
+        <a href="#">Store</a>
+        <a href="#">Blog</a>
+        <a href="#">Help</a>
+        <a href="#">Login</a>
+      </div>
+      <!--end div navbar-->
+
+<div class= "main_store">
+
+<p class= "text_store"> Order your Karma wifi <br>device today!</p>
+
+      </text_store> <!--end div main store-->
+
+<store_image class= "store_image">
+<img src= "img/store-image.jpeg" alt="Lady sitting at her desk, holding a coffee mug" style= "height: 189%">  
+<!--image by neel andrew unsplash-->
+</store_image>
+      </div> <!--end div main store-->
+<form>
+  <fieldset>
+  <label for= "form_firstname">First Name*</label> <br>
+  <input type= "text" name="firstname" id= "form_name" required>
+
+<div class= "container_lastName">
+  <label for= "form_lname" class= "last_name">Last Name*</label>
+  <input type= "text" name="lname" id= "form_name" required>
+  <br>
+  <br>
+</div>
+
+  <label for= "form_address">Address*</label> <br>
+  <input type= "text" name= "address" id= "form_address" class= "address" required>
+ <br>
+ <br>
```

Comment:
Try to use CSS styling rather than `<br>` tags. Flexbox or CSS-grid could help with styling these `<inputs>`